.gitignore

```diff
@@ -27,3 +27,7 @@ docs/src
 _site
 .sass-cache
 .jekyll-metadata
+
+# Junk
+.DS_Store
+
```

Comment:
I think given that it's an OS specific thing, and we don't also ignore things like `thumbs.db`, I would much rather let developers manage their own environment config, and only worry about files that the project generates here.